### DISCUSSION TITLE
在头文件中无法引用库的头文件

### DISCUSSION THREAD

**Discussion by Mano-Liaoyan** [xmake-io/xmake#5219] [Q&A]
我想做一个dll来封装一下mbedTLS, 需要在头文件中引用mbedTLS相关的类，但是似乎xmake中的头文件无法应用安装的包的头文件，在cpp文件中引用时正常的。mbedTLS是xrepo的包。
foo.h
![image](https://github.com/xmake-io/xmake/assets/43143420/48f8adaa-1903-41f2-b39d-6ee707bb597e)
foo.cpp
![image](https://github.com/xmake-io/xmake/assets/43143420/58951e25-01ea-4c5d-a229-19749ba74c25)
xmake.lua
![image](https://github.com/xmake-io/xmake/assets/43143420/c7f837e6-a15d-406e-9bd1-70410c2c0745)

**Comment by waruqi** [comment]
没明白啥问题？不能编译，头文件找不到编译报错？那就贴下 xmake -vD 的 error logs 和完整编译 logs
